packages/@angular/cli/models/webpack-configs/utils.ts

```diff
@@ -14,16 +14,21 @@ const webpackOutputOptions = {
   modules: false,
   reasons: false,
   warnings: true,
-  assets: false, // listing all assets is very noisy when using assets directories
-  version: false
+  errors: true,
+  assets: true, // listing all assets is very noisy when using assets directories
```

Comment:
`assets` should remain false.

Reply:
It's used here: https://github.com/angular/angular-cli/blob/master/packages/%40angular/cli/utilities/stats.ts#L32

The CLI has actually been using the `verbose` option which enables `assets` by default. https://github.com/angular/angular-cli/blob/master/packages/%40angular/cli/tasks/build.ts#L47

I updated the comments to reflect which options are required by the CLI's custom output formatter.

Reply:
I see now, thank you for explaining it to me.